### DISCUSSION TITLE
consumer:connection to broker lost

### DISCUSSION THREAD

**Discussion by fangyuandoit** [celery/celery#7236] [Q&A]
## sometime celery consumer can't receive task And sometime will happen this error that  consumer:connection to broker lost
my properties:
redis==2.1.5
kombu==4.1.0
celery==4.1.0

![901642583730_ pic](https://user-images.githubusercontent.com/12381899/150104908-604f1eb4-4131-4efe-a1da-013c6ffa6420.jpg)

what more information can get from this picture or some solution
thanks  everythone..

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
closing for not following the guideline
